v22.1/backup-architecture.md

```diff
@@ -0,0 +1,101 @@
+
+## Overview
+
+A high-level workflow of a backup:
+
+1. Validate the `BACKUP` statement.
+1. Determine the keys in the [storage layer](architecture/storage-layer.html) to back up.
+1. Write the backup data to the destination storage location.
+1. Write the metadata of the backup job.
+
+The following diagram illustrates the flow from `BACKUP` statement through to a complete backup in cloud storage:
+
+<img src="{{ 'images/v22.1/backup-overview.png' | relative_url }}" alt="A flow diagram representing the process of a backup job from statement through to backup data stored." style="border:0px solid #eee;max-width:100%" />
+
+### Glossary 
+
+Term         | Description
+-------------+-----------------------------
+Coordinator  | The node that claims the job and coordinates the distributed work for the backup job.
+Job record   | Details of the backup job that are defined in the planning phase of the backup.
+Job registry | On each node, this polls the cluster's system jobs table to find jobs to claim.
+Manifest     | A metadata representation of the backup, which is stored with the backup data in the storage location.
+
+## Planning
+
+A backup job begins with a planning phase that validates the general sense of the proposed backup.
+
+Let's take the following statement to start the backup process:
+
+~~~ sql
+BACKUP DATABASE movr INTO 's3://bucket' WITH revision_history;
+~~~
+
+During planning, CockroachDB will verify the [options](backup.html#options) (e.g., `revision_history`) passed in the `BACKUP` statement and check that the user has the [required privileges](backup.html#required-privileges) to take the backup. The tables are identified along with the set of keys to back up. In this example, the backup will identify all the tables within the `movr` database and note that the `revision_history` option is required.
+
+The ultimate aim of the planning phase is to complete all of these checks and write the detail of what the backup job should complete to a _job record_.
+
+Planning happens within a single transaction in which the job record is written and committed as part of the transaction. As a result, [pausing](pause-job.html) and [canceling](cancel-job.html) a backup is not possible during the planning component since it is not yet a job. 
+
+Once the statement has been passed through the planning phase a `job ID` output will return, which represents that the job has started executing.
+
+ ~~~
+        job_id
+----------------------
+  592786066399264769
+(1 row)
+ ~~~
+
+As soon as the detail of the backup is written to the job record, the planning phase’s transaction will commit and the nodes can claim the backup job.
+
+Clusters have system tables that are visible to each of their nodes. One of these tables, the `system.jobs` table, is a list of all adoptable jobs. Each row in this table represents a job, which is described by the job record created in the backup's planning phase.
```

Comment:
mentioned above, but these next couple lines seem like they _might_ be better moved to a general "how jobs work" doc?